cirq-core/cirq/ops/pauli_gates_test.py

```diff
@@ -220,3 +220,24 @@ def test_powers():
     assert isinstance(cirq.X**1, cirq.Pauli)
     assert isinstance(cirq.Y**1, cirq.Pauli)
     assert isinstance(cirq.Z**1, cirq.Pauli)
+
+
+@pytest.mark.parametrize('gate_cls', (cirq.XXPowGate, cirq.YYPowGate, cirq.ZZPowGate))
+@pytest.mark.parametrize('exponent', (0, 0.5, 0.75, 1, 1.5))
+def test_clifford_protocols(gate_cls: type[cirq.EigenGate], exponent: float):
+    gate = gate_cls(exponent=exponent)
+    assert hasattr(gate, '_decompose_into_clifford_with_qubits_')
+    is_clifford = exponent % 2 in (0, 0.5, 1, 1.5)
```

Comment:
NIT - this repeats expression from the tested code.  It would be a bit more robust to set is_clifford in exponent parametrization, ie, use 2 arguments *exponent*, *is_clifford*.

Reply:
added